python/gym_ignition/randomizers/physics/dart.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+from gym_ignition.randomizers.base import physics
+from gym_ignition import scenario_bindings as bindings
+
+
+class DART(physics.PhysicsRandomizer):
+    """
+    Class that configures the Ignition Gazebo physics with the DART physics engine and
+    no randomization.
+    """
```

Comment:
Wouldn't it prevent confusion to call it e.g. `DART_no_rand` in view of other DART-targeted classes indeed introducing randomization?

Reply:
I thought about it and given that there could be multiple types of randomizations I decided to do the opposite. Call this just `DART` and use a longer and more descriptive name to classes that perform randomizations (e.g. `DARTRandomizeGravity`, ...).

Reply:
Ok